app/grandchallenge/components/models.py

```diff
@@ -2030,7 +2030,20 @@ class Meta:
 
     @property
     def animate(self):
-        return self.import_status == self.ImportStatusChoices.STARTED
+        return self.import_status in {
```

Comment:
Why change the states when animation is done? Given how the import task works I think the previous was correct.

Reply:
reverted